test/integration/stepper/verticalStepper.spec.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-env mocha */
+import React from 'react';
+// import ReactDOM from 'react-dom';
+import {mount} from 'enzyme';
+import {assert} from 'chai';
+import getMuiTheme from 'src/styles/getMuiTheme';
+import VerticalLinearStepper from '../fixtures/VerticalLinearStepper';
+
+describe('Vertical Stepper', () => {
+  const muiTheme = getMuiTheme();
+  const mountWithContext = (node) => mount(node, {context: {muiTheme}});
+
+  const wrapper = mountWithContext(
+    <VerticalLinearStepper />
+  );
+  const steps = wrapper.find('Step');
+
+  describe('initial rendering', () => {
+    it('should render a vertical stepper', () => {
+      const stepper = wrapper.find('Stepper');
+      assert.strictEqual(stepper.length, 1, 'there should be a stepper');
+      assert.strictEqual(stepper.prop('orientation'), 'vertical', 'it should be vertical');
+    });
+
+    describe('steps', () => {
+      it('should render 3 steps', () => {
+        assert.strictEqual(steps.length, 3, 'there should be 3 steps');
+      });
+
+      it('should have an active 1st step', () => {
+        assert.ok(steps.at(0).prop('active'), 'should be active');
```

Comment:
Since this is an integration test and we're bundling the transition into the `StepContent` component as a default for the vertical stepper -- I'm thinking that I should implement the prop I mentioned above into `StepContent` so the transition is pluggable and can be tested more easily 👍